specification/iotoperations/resource-manager/Microsoft.IoTOperations/preview/2024-08-15-preview/examples/Instance_CreateOrUpdate_MaximumSet_Gen.json

```diff
@@ -1,11 +1,11 @@
 {
```

Comment:
after making the changes to the other example file, delete this one and then re-run the open api script